### ISSUE TITLE
Dud/Blank Spaces

### ISSUE THREAD

**Issue by Mimic-exe** [PartyPlanner64/PartyPlanner64#126]
Currently the only dud space is the Star Space users can place as many Star Spaces on their boards as possible using the Advanced UI option 
![image](https://user-images.githubusercontent.com/102052961/159319620-da65d01e-3f84-48a7-abca-f18dd48990d7.png)
I would be nice to have the option to make any space a dud space so that users can place their own events on them. 
So for example: if a user uses a dud/blank item space, they can then place whatever custom event on it, so when player lands said Item space, the only event that triggers is a custom event to do what ever it is they need it to.

**Comment by PartyPlanner64** [comment]
I agree it would be nice to do something like this. The game engine hard codes the behavior of certain space types though, so I am not sure how this sort of enhancement could be done.
